poly-synth/src/poly-synth.js

```diff
@@ -52,13 +55,17 @@ class PolySynth {
     this.minDetuneAmount = 0;
     this.maxDetuneAmount = 5;
 
+    this.minQ = 0;
+    this.maxQ = 100;
+
     // The initial values for the parameters are experimentally determined.
     this.parameters_ = {
-      filterCutoff: 440,
-      gainAttack: this.minAttack,
+      gainAttack: 0,
       gainDecay: this.minDecay,
       gainSustain: this.minSustain,
-      gainRelease: this.minRelease,
+      gainRelease: 0.1,
+      filterCutoff: 440,
+      filterQ: this.minQ,
```

Comment:
Q has a pretty profound effect on the sound, and adds a lot of character. I think it's better to leave it "off" initially and let the user increase it if they so desire.